### DISCUSSION TITLE
How can I programmatically disable and enable liveEdit on nuxt-content ?

### DISCUSSION THREAD

**Discussion by azataiot** [nuxt/nuxt#9760] [Questions (Nuxt 2)]
How can I programmatically disable and enable liveEdit ? (So that I can use nuxt-content together with nuxt-auth, allowing only the authenticated people to live edit the content of the page?)

**Comment by kissu** [comment]
You asked the same question on [stackoverflow](https://stackoverflow.com/q/69060519/8816585), meanwhile to help you here or there, we need some details, code or context. Hence why it was closed on Stackoveflow.
